parler/managers.py

```diff
@@ -56,11 +55,11 @@ def _extract_model_params(self, defaults, **kwargs):
                     pass
 
         if django.VERSION < (2, 2):
-            lookup, params = super(TranslatableQuerySet, self)._extract_model_params(defaults, **kwargs)
```

Comment:
Django-2.1 and below reached EOL. Shouldn't we drop this altogether?

Reply:
Sure!